### ISSUE TITLE
Midi  to Mouse position

### ISSUE THREAD

**Issue by Almaper** [cbdevnet/midimonster#132]
Good evening, with a midi input,  is it possible to define a starting position for the mouse cursor?
Thanks

**Comment by Spacelord09** [comment]
Not really. You could set the cursor to a specific point with our programming backends and go from there.
